pyvo/dal/sia2.py

```diff
@@ -90,6 +94,14 @@
 """
 
 
+def __getattr__(name):
+    if name == 'SIA_PARAMETERS_DESC':
+        warnings.warn("The name SIA_PARAMETERS_DESC is deprecated in v1.5 for SIA v2 services, "
+                      "use SIAv2_PARAMETERS_DESC instead.", AstropyDeprecationWarning)
+        return SIAv2_PARAMETERS_DESC
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```

Comment:
Since we're setting a precedent here for future major version changes: Do we want to go mixed-case here (SIAv2_PARAMETERS...)?  I could live with this, but it feels a bit eclectic, and I *think* SIA2_PARAMETERS would work about as well.  And as the short name in Registry discovery, I plan to have sia2 (rather than siav2 or something), too...

Reply:
OK, fair enough about naming the global, and I agree there the SIA2_PARAMETERS... makes sense here (I blindly used the same rename pattern everywhere. 

I still have a slight preference for naming the classes with v2, just as I feel it makes it easier to notice the difference, the 2 would otherwise just hide for me. But it's a slight preference, and I'm happy to change to whatever we come up with.